packages/python/libsql-client/libsql_client/http_driver.py

```diff
@@ -40,6 +70,9 @@ async def batch(self, stmts: List[_RawStmt]) -> List[ResultSet]:
     async def close(self) -> None:
         await self._session.close()
 
+    def queries_url(self):
```

Comment:
Ditto, it would be better to prefix the private method with `_`. Also, perhaps we should add the logic for switching between `/` and `/queries` here?

Reply:
yes this is a placeholder for switching the route, but the queries route is not yet merged, and this will be merged first

Reply:
In that case, perhaps we should merge this PR later? At this point, it's just a noop that introduces extra latency.